CleanProFinder/CleanProFinder.Mobile/Views/ServiceUserStartingPage.xaml.cs

```diff
@@ -0,0 +1,12 @@
+using CleanProFinder.Mobile.ViewModels;
+
+namespace CleanProFinder.Mobile.Views;
+
+public partial class ServiceUserStartingPage : ContentPage
+{
+	public ServiceUserStartingPage(ServiceUserStartingViewModel viewModel)
+	{
+        InitializeComponent();
+        BindingContext = viewModel;
+	}
```

Comment:
Add indentation in ServiceUserStartingPage constructor

Reply:
Fixed